project_task_material_stock/models/project_task.py

```diff
@@ -189,15 +194,13 @@ def _prepare_stock_move(self):
 
     @api.multi
     def create_stock_move(self):
-        pick_type = self.env.ref(
```

Comment:
Maybe you should leave this picking type as fallback.

Reply:
The problem here is that in multi company environment, picking type is assigned even if company haven't a picking type because model stock.picking.type hasn't company_id field, so we have a picking type with locations of another company.

Reply:
maybe we have to do some check in warehouse to avoid this problem.

Reply:
Yeah, but you have now the option to avoid it setting explicitly the picking type here.

Reply:
ok, I understood.